apps/web/docker/Dockerfile

```diff
@@ -1,25 +1,63 @@
-FROM latitudedata/base AS installer
-
-ARG NODE_ENV=development
+FROM latitudedata/base AS base
 
 RUN turbo prune @latitude-data/web --docker
 
-FROM installer AS runner
+FROM base AS prunner
 
-COPY .gitignore .gitignore
-COPY --from=installer /app/out/full .
+# COMMON stage (installer)
+# ------------------------------------------------------
+FROM base AS installer
+COPY --from=prunner /app/out/json/ .
+COPY --from=prunner /app/out/pnpm-lock.yaml ./pnpm-lock.yaml
+COPY --from=prunner /app/out/pnpm-workspace.yaml ./pnpm-workspace.yaml
+COPY --from=prunner /app/out/full/ .
 
-RUN --mount=type=cache,id=pnpm,target=/pnpm/store \
-    if [ "$NODE_ENV" = "production" ]; then \
-        pnpm install --prod --frozen-lockfile; \
-    else \
-        pnpm install; \
-    fi
+RUN --mount=type=cache,id=pnpm,target=/pnpm/store pnpm install \
+  --frozen-lockfile \
+  --filter "@latitude-data/web..."
```

Comment:
do you need this filter? prune is already setting up the dependencies so that npm install only installs the deps for web afaik

Reply:
What about `api` app? How pnpm knows it only have to install `web` and its dependencies?

Reply:
because you run the prune for @latitude-data/api previously afaik

Reply:
Ok, I'll remove and try

Reply:
It does try to build api without this file
![image](https://github.com/latitude-dev/latitude-llm/assets/49499/0f3cf268-cd8b-4498-b31c-a00938959991)

Reply:
Without this install fails. And even if it didn't failed it does not hurt. Does it?